### ISSUE TITLE
Contribution

### ISSUE THREAD

**Issue by Sheko100** [DoaaAbdallah1/Palestine.github.io#1]
Hello Doaa,

I saw your post on LinkedIn, and you mentioned that the project is open for contribution, so I would love to contribute to a project as valuable as that. So what is still missing that I can contribute to? I have experience with both front and back ends.

**Comment by DoaaAbdallah1** [comment]
Hello Shaker,

I am happy because you want to join us. All I want is your github account,
and if possible, a WhatsApp number for communication.

في الخميس، ١٤ ديسمبر ٢٠٢٣, ٦:٥٤ م Shaker ***@***.***> كتب:

**Comment by Sheko100** [comment]
Yeah sure. This is my GitHub profile https://github.com/Sheko100 and this is my number on WhatsApp 01227346889
